frontend/test/metabase/scenarios/question/reproductions/28221-missing-custom-field-metadata.cy.spec.js

```diff
@@ -0,0 +1,50 @@
+import { restore } from "__support__/e2e/helpers";
+
+import { SAMPLE_DATABASE } from "__support__/e2e/cypress_sample_database";
+
+const { PRODUCTS_ID, PRODUCTS, ORDERS_ID, ORDERS } = SAMPLE_DATABASE;
+
+describe("issue 28221", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should be able to select see notebook view even if a question custom field metadata is missing#27462", () => {
+    const questionName = "Reproduce 28221";
+    const customFieldName = "Non-existing field";
+    const questionDetails = {
+      name: questionName,
+      query: {
+        "source-table": ORDERS_ID,
+        joins: [
+          {
+            fields: "all",
+            "source-table": PRODUCTS_ID,
+            condition: [
+              "=",
+              ["field", ORDERS.PRODUCT_ID, null],
+              ["field", PRODUCTS.ID, { "join-alias": "Products" }],
+            ],
+            alias: "Products",
+          },
+        ],
+        expressions: {
+          [customFieldName]: ["field", 9999, null],
+        },
+      },
+    };
+
+    cy.createQuestion(questionDetails, { idAlias: "questionId" }).then(
```

Comment:
Are you using `idAlias` here? I think it's only used with `wrapId: true`

Reply:
Fixed by removing the option altogether.